### PR TITLE
create incompressibleNavierStokesSolver (2nd PR)

### PR DESCRIPTION
**Pull request by feilin-flexcompute**
use 2 class to define compressible and incompressible NS solvers respectively

### CODE REVIEW

File: flow360/component/flow360_params/solvers.py [hunk 1]
```diff
@@ -186,6 +186,45 @@ def _get_optional_objects(cls) -> List[str]:
...
+    """
+
+    pressure_correction_solver: Optional[PressureCorrectionSolver] = pd.Field(
+        alias="pressureCorrectionSolver"
+    )
+    linear_solver: Optional[LinearSolver] = pd.Field(
+        LinearSolver(max_iterations=30), alias="linearSolver"
+    )
+    update_jacobian_frequency: Optional[PositiveInt] = pd.Field(
+        4, alias="updateJacobianFrequency", displayed="Update Jacobian frequency"
```
[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply flow360/component/flow360_params/solvers.py]: removed.

[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply flow360/component/flow360_params/solvers.py:223]: Also, the doc string of the incompressible NS solver has been updated to use camel case, similar to original NS solver

[the remark this answers is not in the document]
Comment (maciej-flexcompute) [reply flow360/component/flow360_params/solvers.py:223]: I see, lets use snake case for docstring

[the remark this answers is not in the document]
Comment (feilin-flexcompute) [reply flow360/component/flow360_params/solvers.py:223]: done.

**Pull request merged**